docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Explore code with the syntax visualizer
```

Comment:
For SEO, something like "explore code with the Roslyn syntax visualizer in Visual Studio".